test/unit/test_asyncify.py

```diff
@@ -26,8 +26,9 @@ def test_asyncify_pure_wasm(self):
     run_process(WASM_OPT + [self.input_path('asyncify-pure.wast'), '--asyncify', '-o', 'a.wasm'])
     run_process(WASM_DIS + ['a.wasm', '-o', 'a.wast'])
     output = run_process(WASM_SHELL + ['a.wast'], capture_output=True).stdout
-    with open(self.input_path('asyncify-pure.txt')) as f:
-      self.assertEqual(f.read(), output)
+    with open(self.input_path('asyncify-pure.txt'), 'r') as f:
+      temp = f.read()
+      self.assertEqual(temp, output)
```

Comment:
Why was this `temp` variable introduced? It's not necessary IMO.

Reply:
That was for debugging, fixing, thanks.